EasyClangComplete.py

```diff
@@ -136,20 +136,20 @@ def on_activated_async(self, view):
             # All is taken care of. The view is built if needed.
             self.view_config_manager.load_for_view(view, settings)
 
-    def on_selection_modified(self, view):
-        """Called when selection is modified. Executed in gui thread.
-
-        Args:
-            view (sublime.View): current view
-        """
-        if Tools.is_valid_view(view):
-            (row, _) = SublBridge.cursor_pos(view)
-            view_config = self.view_config_manager.get_from_cache(view)
-            if not view_config:
-                return
-            if not view_config.completer:
-                return
-            view_config.completer.error_vis.show_popup_if_needed(view, row)
+    # def on_selection_modified(self, view):
+    #     """Called when selection is modified. Executed in gui thread.
+
+    #     Args:
+    #         view (sublime.View): current view
+    #     """
+    #     if Tools.is_valid_view(view):
+    #         (row, _) = SublBridge.cursor_pos(view)
+    #         view_config = self.view_config_manager.get_from_cache(view)
+    #         if not view_config:
+    #             return
+    #         if not view_config.completer:
+    #             return
+    #         view_config.completer.error_vis.show_popup_if_needed(view, row)
```

Comment:
Remove unused code instead of disabling.
But I'm not sure if it's a good idea to just remove that functionality and force phantoms. Phantoms can be annoying as they make lines jump up and down. You should maybe consider making it an option (with default being either one or the other - I don't have strong opinion on that).